arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -23,178 +23,227 @@
         ],
         "description": "Gets a failover group.",
         "operationId": "FailoverGroups_Get",
-        "x-ms-examples": {
-          "Get failover group": {
-            "$ref": "../examples/FailoverGroupGet.json"
-          }
-        },
         "parameters": [
           {
-            "$ref": "#/parameters/ApiVersionParameter"
-          },
-          {
             "$ref": "#/parameters/SubscriptionIdParameter"
           },
           {
             "$ref": "#/parameters/ResourceGroupParameter"
           },
           {
-            "$ref": "#/parameters/ServerParameter"
+            "name": "serverName",
+            "in": "path",
+            "description": "The name of the server containing the failover group.",
+            "required": true,
+            "type": "string"
           },
           {
-            "$ref": "#/parameters/FailoverGroupParameter"
+            "name": "failoverGroupName",
+            "in": "path",
+            "description": "The name of the failover group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
```

Comment:
Ok, this is doable. Do you have a discussion I can link to as justification in my Swagger generator source code?